src/commons/modifiers.ts

```diff
@@ -100,30 +103,34 @@ export type ContainerModifiers = AlignmentModifiers &
   BorderRadiusModifiers &
   BackgroundColorModifier;
 
+export function updateColorModifiers() {
```

Comment:
If you'll think about it, maybe it's better that the Colors class will maintain this logic and provide a cached list of colors keys. 
By moving this logic to the Colors class, you can also update the cached list whenever loadColors is being called - so you don't need to worry about it as a consumer. 
WDYT?

Reply:
I wondered about it.. you can see my note in this PR description: https://github.com/wix-private/wix-react-native-ui-lib/pull/2473

Reply:
Oh ok, I didn't see the private PR yet. 
I think we should call and keep it in the Colors class, what are the cons for doing that?

Reply:
The only disadvantage I can think of is that it'll cause an overhead in the `loadColors` method in a case that the user calls it many times in his app..  but I guess it's a rare case and we can ignore it
I'll update it :)